corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/util/util.js

```diff
@@ -0,0 +1,24 @@
+function Util() {
+}
+
+Util.getQueryParams = function (qs) {
+    qs = qs.split("+").join(" ");
+
+    var params = [], tokens,
+        re = /[?&]?([^=]+)=([^&]*)/g;
+
+    while (tokens = re.exec(qs)) {
```

Comment:
I hadn't seen this pattern before, but yeah, looks like it's syntactically valid.  This:
```javascript
while (tokens = re.exec(qs)) {
    params.push({...});
}
```
Is basically shorthand for
```javascript
var tokens = re.exec(qs);
while (tokens) {
    params.push({...});
    tokens = re.exec(qs);
}
```
I think the reason why this lint rule exists is because it's possible what you meant to do was:
```javascript
while (tokens === re.exec(qs)) {
    params.push({...});
}
```
I think it's fine to ignore.